n2y/utils.py

```diff
@@ -172,6 +172,15 @@ def sanitize_filename(filename):
     return s
 
 
+def slugify(string):
+    """Turn a string into a slug."""
+    s = str(string).strip().replace(" ", "-")
```

Comment:
We'll want the "slugify" to make the text lower case too. It should work like https://docs.djangoproject.com/en/5.0/ref/utils/#django.utils.text.slugify

You could probably even grab the code from Django.

Note slugify is a different function from the sanatize_filename. sanitize_filename should still be used in all of the places it's currently being used in. Also, you don't need to check for "." or ".." in the slugify function.